rotkehlchen/history/events/structures/base.py

```diff
@@ -190,11 +191,13 @@ def _serialize_base_tuple_for_db(self) -> tuple[str, str, HISTORY_EVENT_DB_TUPLE
             (
                 'history_events(entry_type, event_identifier, sequence_index,'
                 'timestamp, location, location_label, asset, amount, notes,'
-                'type, subtype, extra_data) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)'
+                'type, subtype, extra_data, ignored) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, '
+                "(CASE WHEN EXISTS (SELECT 1 FROM multisettings WHERE name = 'ignored_asset' AND value = ?) THEN 1 ELSE 0 END))"  # noqa: E501
```

Comment:
wait why are you doing this? `ignored` is now on the table. Why would you complicate each event serialization like this. This makes no sense.

I think you probably mean to do this on addition and editing? But then do it only at editing and addition as you may want to call serialize at other points, where ignored is already properly populated